pyqt-configure

```diff
@@ -1,9 +1,12 @@
 #!/bin/sh
 
+PYTHON_VERSION=$(${PYTHON} -c 'import sys; print(sys.version[:3])')
+
 ${PYTHON} configure.py \
     --assume-shared \
     --concatenate \
     --confirm-license \
+    --destdir=/app/lib/python${PYTHON_VERSION}/site-packages \
```

Comment:
That's a pretty terrible hardcoding here. :confused: 

You're assuming that it's a path added to Python's sys.path in the runtime.

I wish you hadn't merged that.

Reply:
We're already doing the same hardcoding thing for sip anyway... I couldn't find another way of doing it.